samples/features/security/ledger/azure-sql-database/setup/CreateDatabaseSchema.sql

```diff
@@ -0,0 +1,194 @@
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[Employees](
+	[EmployeeID] [int] IDENTITY(1,1) NOT NULL,
+	[SSN] [char](11) NOT NULL,
+	[FirstName] [nvarchar](50) NOT NULL,
+	[LastName] [nvarchar](50) NOT NULL,
+	[Salary] [money] NOT NULL
+	)
+WITH 
+(
+  SYSTEM_VERSIONING = ON,
+  LEDGER = ON
+); 
+GO
+
+
+CREATE TABLE [dbo].[AuditEvents](
+	[Timestamp] [Datetime2] NOT NULL DEFAULT (GETDATE()),
+	[UserName] [nvarchar](255) NOT NULL,
+	[Query] [nvarchar](4000) NOT NULL
+	)
+WITH (LEDGER = ON (APPEND_ONLY = ON));
+GO
+
+CREATE TABLE [dbo].[LedgerVerifications](
+	[Timestamp] [Datetime2] NOT NULL DEFAULT (GETDATE()),
+	[DigestLocations] [nvarchar](max) NOT NULL,
+	[Result] [nvarchar](255) NOT NULL,
+)
+WITH (LEDGER = ON (APPEND_ONLY = ON));
+GO
+
+CREATE PROCEDURE [dbo].[GetEmployeeLedgerEntries]
+AS 
+SET NOCOUNT ON 
```

Comment:
The syntax will work fine as is. But please move SET NOCOUNT ON into the BEGIN...END block. It is little bit odd to use optional BEGIN END block for proc body but not put the statements inside it.